build.gradle

```diff
@@ -30,7 +30,7 @@ war {
 sourceSets.main.java.srcDirs += "src/main/generated"
 
 dependencies {
-    def jopaVersion = '2.0.0'
```

Comment:
@kostobog good that you updated JOPA but I have little worries about it, that the update is not complete:

See [JOPA release notes](https://github.com/kbss-cvut/jopa/releases/tag/v2.0.2):
- Dependency updates: RDF4J 5.0.0.
  - Update note: Still compatible with server 4 and, with the [right configuration](https://github.com/kbss-cvut/jopa/releases/tag/v1.0.0),
    also 3. However, if you are passing repository configuration file to JOPA, make sure it uses the [new configuration vocabulary](https://rdf4j.org/documentation/reference/configuration/).

Reply:
@blcham 
I checked the release notes. Use graphdb 10.x which uses rdf4j 4.x both in fta-fmea-ui and distribution docker-compose. It should be compatible without additional changes.

Reply:
thanks.